tests/templates/test_subroutines/test_trotter.py

```diff
@@ -211,6 +213,93 @@
     }
 )
 
+test_resources_data = {
```

Comment:
You can bring it close to the class that uses it.

Reply:
The other data tuples are also defined at the top, so I leave it here for consistency.